src/session.c

```diff
@@ -387,10 +387,49 @@ CK_SLOT_ID p11prov_session_slotid(P11PROV_SESSION *session)
     return session->slotid;
 }
 
+static int p11prov_session_get_pin(struct p11prov_slot *slot, char *cb_pin,
```

Comment:
Can you rename this one to p11prov_session_prompt_for_pin() ?

Reply:
Done